src/uct/ib/efa/ib_efa_dv.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_IB_EFA_DV_H_
+#define UCT_IB_EFA_DV_H_
+
+#ifndef UCT_IB_EFA_H_
+# error "Never include <uct/ib/efa/ib_efa_dv.h> directly; use <uct/ib/efa/ib_efa.h> instead."
```

Comment:
why?

Reply:
I was trying to follow the same structure we have for mlx5. It is a question for myself too: why do we have both ib_mlx5_dv.h and ib_mlx5.h?

Reply:
it's mostly for historical reasons. for efa, would suggest having either ib_efa.h or ib_efadv.h

Reply:
Will keep ib_efa.h only.